src/components/Profile/components/MembershipsList/components/MembershipInfoCard/index.jsx

```diff
@@ -15,6 +16,9 @@ const MembershipInfoCard = ({ myProf, membership, onTogglePrivacy }) => {
           className={`gc360_text_link ${
             showPrivate ? styles.private_membership : styles.public_membership
           }`}
+          style={{
+            color: newTheme.colorSchemes.light.palette.primary[300],
+          }}
```

Comment:
We definitely shouldn't add **new** inline styles - and also this one is hard-coding the wrong value. I know you probably didn't add this, but it needs to be removed before we merge.

Reply:
Fair enough, yeah, I think this was from before I was working on it, so I'll revert it until we get to this file for real